extension/chrome/src/js/background.js

```diff
@@ -174,8 +174,8 @@ function retrieveActivities() {
 
 		Promise.all([reviewersPromised, authorPromised]).then(function([reviewersResult, authorResult]) {
 			if (reviewersResult.errors) { throw reviewersResult.errors }
-			if (authorResult.errors) { throw authorResult.errors }
-			const allPR = reviewersResult.concat(authorResult)
+            if (authorResult.errors) { throw authorResult.errors }
```

Comment:
Can you, please, fix the indents, for better readability? Maybe caused by tabs vs spaces.

Reply:
Oh yep. I'm spaces. I'll correct it.